### PR TITLE
Translate "What are third-party origin trials?" into Japanese

### PR DESCRIPTION
**Pull request by yoichiro**
I have translated the [What are third-party origin trials? - Chrome Developers](https://developer.chrome.com/blog/third-party-origin-trials/) article into Japanese.

Post: [What are third-party origin trials? - Chrome Developers](https://developer.chrome.com/blog/third-party-origin-trials/)
Translator: @yoichiro
Reviewer: @agektmr

### CODE REVIEW

**Comment by yoichiro** [comment]
@chybie @samdutton @agektmr I have just translated the [What are third-party origin trials? - Chrome Developers](https://developer.chrome.com/blog/third-party-origin-trials/) into Japanese on this pull request. Could you review this pull request? Thank you in advance!

**Comment by samdutton** [comment]
@jpchase fyi

File: site/ja/blog/third-party-origin-trials/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+provides more general information about how to participate in Chrome origin trials. -->
+
+サードパーティオリジントライアルは、すべての機能で利用できるわけではありません。Chromeは、サードパーティのサイトにコードを埋め込むことが一般的なユースケースとなる機能に対してのみ、サードパーティオリジントライアルオプションを利用できるようにします。 [Chrome のオリジントライアル入門](https://developers.chrome.com/origintrials/) には、Chrome オリジントライアルへの参加方法に関するより一般的な情報が記載されています。
+
+<!-- If you participate in an origin trial as a third-party provider, it will be your responsibility to
+notify and set expectations with any partners or customers whose sites you intend to include in the
+origin trial. Experimental features may cause unexpected issues and browser vendors may not be able
+to provide troubleshooting support. -->
+
+サードパーティプロバイダとしてオリジントライアルに参加する場合、オリジントライアルに含める予定のサイトのパートナーまたは顧客に通知し、見込みを設定するのは、あなたの責任となります。実験的な機能により予期しない問題が発生する可能性があります。そして、ブラウザベンダは、トラブルシューティングサポートを提供できないかもしれません。
```
Comment (lacolaco) [inline site/ja/blog/third-party-origin-trials/index.md:49]: set expectationsの訳は難しいですね　対案は浮かばないですがしっくりこない感覚があります

File: site/ja/blog/third-party-origin-trials/index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+1. Click the Register button to submit your request.
+1. Your third-party token will be issued immediately, unless further review of the request is
+   required. (Depending on the trial, token requests may require review.)
+1. If review is required, you'll be notified by email when the review is complete and your
+   third-party token is ready.
+-->
+
+1. [進行中のトライアルの一覧](https://developers.chrome.com/origintrials/#/trials/active) からトライアルを選択します。
+1. トライアルの登録ページにて、サードパーティトークンを要求するためのオプションが利用可能になっていれば、それを有効にします。
+1. サードパーティトークンの使用を制限するための選択肢を 1 つを選択します:
```
Comment (lacolaco) [inline site/ja/blog/third-party-origin-trials/index.md:93]:
```suggestion
1. サードパーティトークンの使用を制限するための選択肢の 1 つを選択します:
```

**Review by lacolaco — COMMENTED** [review]
@yoichiro レビューしました！

File: site/ja/blog/third-party-origin-trials/index.md (hunk 1 above)
Comment (agektmr) [reply site/ja/blog/third-party-origin-trials/index.md:49]: 「期待値を設定する」はどうですか？

File: site/ja/blog/third-party-origin-trials/index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+testing scripts that are embedded across a number of sites. -->
+
+オリジントライアルは通常、ファーストパーティベースでのみ利用でき、登録された単一の [オリジン](https://web.dev/same-site-same-origin/#origin) でのみ機能します。もし開発者がコンテンツが埋め込まれている他のオリジンで実験的な機能をテストしたい場合は、それらのオリジンをすべてオリジントライアルに登録しなければならず、それぞれに一意のトライアルトークンが必要になります。これは、多数のサイトに埋め込まれているスクリプトをテストすることを考えると、スケーラブルなアプローチとは言えません。
+
+<!-- Third-party origin trials make it possible for providers of embedded content to try out a new
+feature across multiple sites. -->
+
+サードパーティオリジントライアルにより、埋め込みコンテンツのプロバイダは、複数のサイトを横断して新機能を試せるようになります。
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/fCachIuiBjh3XPo10CrN.png", alt="Diagram showing how third-party origin trials enable a single registration token to be used across multiple origins", width="800", height="400" %}
```
Comment (agektmr) [inline site/ja/blog/third-party-origin-trials/index.md:35]: Translate the `alt` content as well?

File: site/ja/blog/third-party-origin-trials/index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+potential for overuse or abuse of experimental features, so a "trusted tester" approach is more
+appropriate. The greater reach of third-party origin trials requires additional scrutiny and
+additional responsibility for web developers that participate as third-party providers. Requests to
+enable a third-party origin trial may be reviewed in order to avoid problematic third-party scripts
+affecting multiple sites. The Origin Trials Developer Guide explains the
+[approval process](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#18-how-can-i-enable-an-experimental-feature-as-embedded-content-on-different-domains). -->
+
+サードパーティオリジントライアルをサポートすることで、より幅広く参加してもらうことが可能となりますが、実験的な機能の過度の使用または誤用の可能性も高まるため、"trusted tester" アプローチの方が適切です。 サードパーティオリジントライアルの範囲を拡大するには、サードパーティプロバイダとして参加するウェブ開発者に対して、追加の精査と責任が求められます。複数のサイトに影響を与える問題のあるサードパーティのスクリプトを回避するために、サードパーティのオリジントライアルを有効にするリクエストはレビューが必要となる場合があります。オリジントライアル開発者ガイドにて、 [承認プロセス](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#18-how-can-i-enable-an-experimental-feature-as-embedded-content-on-different-domains) が説明されています。
```
Comment (agektmr) [inline site/ja/blog/third-party-origin-trials/index.md:60]:
```suggestion
サードパーティオリジントライアルをサポートすることで、より幅広い参加が期待できますが、実験的な機能の過度の使用または誤用の可能性も高まるため、"トラステッドテスター" アプローチの方が適切です。 サードパーティオリジントライアルによるより広い影響範囲では、サードパーティプロバイダとして参加するウェブ開発者の、さらなる監視と責任が求められます。問題のあるサードパーティのスクリプトが複数のサイトに影響を与えるのを回避するため、サードパーティオリジントライアルを有効にするリクエストは、レビューが必要となる場合があります。オリジントライアル開発者ガイドにて、 [承認プロセス](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#18-how-can-i-enable-an-experimental-feature-as-embedded-content-on-different-domains) が説明されています。
```

File: site/ja/blog/third-party-origin-trials/index.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+1. トライアルの登録ページにて、サードパーティトークンを要求するためのオプションが利用可能になっていれば、それを有効にします。
+1. サードパーティトークンの使用を制限するための選択肢を 1 つを選択します:
+   1. Standard Limit: これは、 [Chrome ページ読み込みの 0.5%](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#3-what-happens-if-a-large-site-such-as-a-google-service-starts-depending-on-an-experimental-feature) に適用される通常の制限です。
+   1. User Subset: 有効なサードパーティトークンが提供されている場合でも、Chrome ユーザーの数パーセントが常にトライアルから除外されます。除外率はトライアルごとに異なります（または適用されないかもしれない）が、通常は 5% 未満です。
+1. Register ボタンをクリックして、リクエストを送信します。
+1. リクエストに対してさらにレビューが求められない限り、サードパーティトークンはすぐに発行されます。（トライアルによっては、トークンリクエストのレビューが必要になる場合があります。）
+1. レビューが必要だった場合は、レビューが完了し、サードパーティトークンの準備ができたときに、メールで通知されます。
+
+   <figure class="w-figure">
+     {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/r07Zb0QoHnlgiItETR6q.png", alt="Chrome origin trials registration page for the Conversion Measurement API, with third-party matching checkbox selected.", width="800", height="618" %}
```
Comment (agektmr) [inline site/ja/blog/third-party-origin-trials/index.md:101]: Translate the `alt` part?

File: site/ja/blog/third-party-origin-trials/index.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+     <figcaption class="w-figcaption">Registration page for the Conversion Measurement trial.</figcaption>
```
Comment (agektmr) [inline site/ja/blog/third-party-origin-trials/index.md:102]: Translate the content of `figcaption`?

File: site/ja/blog/third-party-origin-trials/index.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+<!-- If you're registering for a third-party origin trial and have feedback to share on the process or
+ideas on how we can improve it, [create an
+issue](https://github.com/GoogleChrome/OriginTrials/issues/new) on the Origin Trials GitHub code
+repo. -->
+
+サードパーティオリジントライアルに登録していて、それを改善する方法についてのアイデアや手順といったフィードバックを報告したいときは、オリジントライアル GitHub コードリポジトリに [Issue を作ってください](https://github.com/GoogleChrome/OriginTrials/issues/new) 。
+
+## 参考情報
+
+-  [Getting started with Chrome's origin trials](/blog/origin-trials/)
```
Comment (agektmr) [inline site/ja/blog/third-party-origin-trials/index.md:118]:
```suggestion
-  [Chrome のオリジントライアル入門](/ja/blog/origin-trials/)
```

**Review by agektmr — CHANGES_REQUESTED** [review]
Added some comments.

File: site/ja/blog/third-party-origin-trials/index.md (hunk 1 above)
Comment (yoichiro) [reply site/ja/blog/third-party-origin-trials/index.md]: d341320 で「期待値」に変更しました。

File: site/ja/blog/third-party-origin-trials/index.md (hunk 3 above)
Comment (yoichiro) [reply site/ja/blog/third-party-origin-trials/index.md]: It has been translated at c377e4c.

File: site/ja/blog/third-party-origin-trials/index.md (hunk 5 above)
Comment (yoichiro) [reply site/ja/blog/third-party-origin-trials/index.md]: It has been translated at c377e4c.

File: site/ja/blog/third-party-origin-trials/index.md (hunk 6 above)
Comment (yoichiro) [reply site/ja/blog/third-party-origin-trials/index.md]: It has been translated at c377e4c.

**Comment by yoichiro** [comment]
@agektmr @lacolaco Thank you for the reviewing! I modified them according to your mentions. Could you review them again?

**Review by agektmr — APPROVED** [review]
LGTM

**Pull request merged**